packages/material-ui/src/Backdrop/Backdrop.js

```diff
@@ -9,6 +9,9 @@ export const styles = {
   root: {
     zIndex: -1,
     position: 'fixed',
+    display: 'flex',
```

Comment:
Why not used Grid as a container?

Reply:
The idea is to come with a better default. I would expect more people to want to center the child.

Reply:
do you agree with me?

Reply:
I think what @oliviertassinari meant was that a reasonable default would be to just have a center aligned child. In that case, I don't think Grid is necessary with this use case.

Reply:
Exactly, it's even better if the usage of the Grid is not required.

Reply:
@dominictwlee @oliviertassinari All right.